.cirrus.yml

```diff
@@ -4,10 +4,10 @@ env:
   # Specific warnings can be disabled with -Wno-error=foo.
   # -pedantic-errors is not equivalent to -Werror=pedantic and thus not implied by -Werror according to the GCC manual.
   WERROR_CFLAGS: -Werror -pedantic-errors
-  MAKEFLAGS: -j2
+  MAKEFLAGS: -j4
```

Comment:
Does the CI server actually have 4 CPUs?

Reply:
We set `cpu: 1` or `cpu: 2` depending on the task. But the `greedy` flag will occasionally give us more CPUs when available. Increasing this to `-j4` makes sure we actually can make use of this.